projects/github.com/maxbrunsfeld/counterfeiter/package.yml

```diff
@@ -0,0 +1,29 @@
+distributable:
+  url: https://github.com/maxbrunsfeld/counterfeiter/archive/v{{version}}.tar.gz
+  strip-components: 1
+
+versions:
+  github: maxbrunsfeld/counterfeiter
+
+build:
+  dependencies:
+    tea.xyz/gx/cc: c99
+    tea.xyz/gx/make: '*'
+    go.dev: '*'
+  script: go build $ARGS
+  env:
+    ARGS:
+      - -trimpath
+      - -o={{prefix}}/bin/counterfeiter
+
+provides:
+  - bin/counterfeiter
+
+test:
+  dependencies:
+    go.dev: '*'
+  env:
+    GOROOT: "{{deps.go.dev.prefix}}"
+  script: |
+    counterfeiter -p os 2>&1
+    test -e osshim
```

Comment:
Does this really require `go` to pass its tests, but doesn't require `go` to run otherwise?

Reply:
Yes, you're right, it's my mistake. Added dependency and env in last commit